docs/BasicCommunication/OnEventChanged/index.md

```diff
@@ -16,22 +16,22 @@ SDL uses `OnEventChanged` notification to appropriately manage the hmiLevel and
 #### PHONE_CALL
 
 !!! MUST
-1. Send notification with appropriate parameter value when active call on HMI has been started/ended.
+1. Change audioStreamingState to NOT_AUDIBLE for all apps (keep hmiLevel) when active call on HMI has been started.
 
-2. Send [SDL.OnAppDeactivated](../../sdl/onappdeactivated) to the active app when the phone call is started.
-
-3. Resume the applications to their original state prior to the phone call event in the HMI when the event ends (see note below).
+2. Send notification with appropriate parameter value when the event ends.
 !!!
 
 !!! NOTE
-SDL does not send BC.ActivateApp or BC.OnResumeAudioSource to HMI after the phone call is ended.
+- SDL does not send BC.ActivateApp or BC.OnResumeAudioSource to HMI after the phone call is ended.
+
+- If HU wants to switch the screen (HMIStatus) during PHONE_CALL, they can use API `BC.OnAppDeactivated (AppID)` and `BC.OnAppActivated (AppID)`.
```

Comment:
```suggestion
If the HMI wants to leave the app screen when the PHONE_CALL event occurs, the app's HMI state can be changed accordingly using the `BC.OnAppDeactivated(appID)` and `BC.OnAppActivated(appID)` notifications.
```

Reply:
We think there is no problem to modify according to your suggestion.